packages/common/src/components/utils/Modal/ModalBase.tsx

```diff
@@ -24,9 +28,42 @@ interface Props {
   buttons: React.ReactNode
   content: React.ReactNode
   enableCross?: boolean
+  largeWidth?: boolean
 }
 
-const ModalBase: React.FC<Props> = ({ open, handleClose, title, buttons, content, enableCross }) => {
+const ModalBase: React.FC<Props> = ({ open, handleClose, title, buttons, content, enableCross, largeWidth }) => {
+  const ModalContent = () => (
+    <>
+      {enableCross && (
+        <button type="button" aria-label="Close modal" onClick={handleClose} className="ic-modal__close ic-svg-icon">
+          <svg focusable="false" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+            <path
+              fill="currentColor"
+              d="M12 10.733l5.07-5.07c.35-.35.917-.35 1.267 0 .35.35.35.917 0 1.267L13.267 12l5.07 5.07c.35.35.35.917 0 1.267-.35.35-.917.35-1.267 0L12 13.267l-5.07 5.07c-.35.35-.917.35-1.267 0-.35-.35-.35-.917 0-1.267l5.07-5.07-5.07-5.07c-.35-.35-.35-.917 0-1.267.35-.35.917-.35 1.267 0l5.07 5.07z"
+              transform="translate(-994 -650) translate(410 637) translate(584 13)"
+            />
+          </svg>
+        </button>
+      )}
+      <div className="ic-modal__head" id="demo-modal-content">
+        <h3 id="dialog-title">{title}</h3>
+      </div>
+      <ModalContentWrapper className="ic-modal__body ic-text">{content}</ModalContentWrapper>
+      <div className="ic-button-group ic-button-group--right">{buttons}</div>
+    </>
+  )
+
+  const getText = () =>
+    largeWidth ? (
```

Comment:
Instead of doing this you could send the prop for large width to the styled component and apply the style there. By doing this you don't have to change the component modalbase more than sending a prop to WrapText.